.travis.yml

```diff
@@ -1,18 +1,11 @@
-osx_image: xcode7.1
+osx_image: xcode7.2
 language: objective-c
-before_install:
-  - brew update
-  - brew outdated xctool || brew upgrade xctool
```

Comment:
This was just to make sure the latest version of `xctool` was installed. Before the Travis images weren't keeping up with the updates very frequently.